core/src/main/java/nu/yona/server/properties/YonaProperties.java

```diff
@@ -45,6 +45,8 @@
 
 	private int maxUsers;
 
+	private boolean whitelistEnabled;
```

Comment:
We don't need this. Combining ``maxUsers`` with the white list feature is useless, so we can disable the white list feature when ``maxUsers`` is ``true``, and set ``maxUsers`` to a reasonably big number by default.